LEGO1/mxatomid.h

```diff
@@ -17,6 +23,11 @@ class MxAtomId
     this->m_internal = 0;
   }
 
+  inline MxBool operator ==(const MxAtomId &other) const
+  {
+      return this->m_internal == other.m_internal;
```

Comment:
The 4-space indents are a bit incosistent. If we decide to use 4-space indents we should probably make it a rule and adjust everything else at the same time (I don't really have an opinion on the indents, all I care about would be consistency)

Reply:
I forgot about the 2-space indentation.
https://github.com/isledecomp/isle/pull/46 adds an .editorconfig that should fix this (at least for my ide)